news/2017-09-15-mapping-with-rewards-returns.md

```diff
@@ -0,0 +1,83 @@
+---
+layout: post
+title: "Mapping With Rewards Returns"
+date: 2017-09-15 12:00:00 +0000
+---
+
+New and improved, the open-form **Mapping With Rewards** mapping contests are back! Map a selection of [Featured Artist](https://osu.ppy.sh/beatmaps/artists) tracks for your shot at a variety of prizes, from osu!supporter, custom profile badges, and your submission earning a special spot in the game!
+
+![](https://assets.ppy.sh/media/generic-header.png)
+
+**Mapping With Rewards** is a more open-ended approach to mapping contests first floated in [September of 2015](https://blog.ppy.sh/post/129936251068/mapping-with-rewards-sep-2015) with a simple premise: we offer a bounty for the best beatmap set of a [Featured Artist](https://osu.ppy.sh/beatmaps/artists) song, and a big timeframe to complete it in. 
+
+Casual yet competitive at the same time!
+
+With its return, Mapping With Rewards' simple premise now covers **all** gamemodes instead of just one!
+
+## How does this all work?
+
+It's super simple. We put out some songs, and from the moment this announcement goes live, all of these songs now have a *bounty* on them.
```

Comment:
"super" is super unprofessional. Generally, the "it's simple" comment is not really needed.

Also remove "now" from "all of these songs now have a bounty on them".